tests/next_tests/unit_tests/embedded_tests/test_nd_array_field.py

```diff
@@ -125,3 +131,68 @@ def fma(a: common.Field, b: common.Field, c: common.Field, /) -> common.Field:
 
     result = fma(field_inp_a, field_inp_b, field_inp_c)
     assert np.allclose(result.ndarray, expected)
+
+
+@dataclasses.dataclass
+class FieldMock:
+    domain: common.Domain
+    array_ns = np
```

Comment:
sorry, I have another idea that doesn't require this mock, see signature proposal for _get_slices... function